slave/executors.py

```diff
@@ -134,6 +134,9 @@ def execute(self, benchmark, env, args, prefs):
         # delete profile
         runner.rm("profile/")
 
+        # Force stylo off.
+        prefs["layout.css.servo.enabled"] = False
```

Comment:
Can you put it next to the other Firefox default prefs, for consistency, please? (in `configs.py`)

Agreed using an external file or external service would be even better.